mat/eigen.go

```diff
@@ -156,6 +180,25 @@ func (e *EigenSym) VectorsTo(dst *Dense) {
 	dst.Copy(e.vectors)
 }
 
+// RawQ returns the orthogonal matrix Q from the spectral factorization of the
+// original matrix A
+//
+//	A = Q * Λ * Qᵀ
+//
+// The columns of Q contain the eigenvectors of A.
+//
+// If the returned matrix is modified, the factorization is invalid and should
+// not be used.
+//
+// If the receiver does not contain a successful factorization or eigenvectors
+// not computed, RawU will return nil.
+func (e *EigenSym) RawQ() Matrix {
```

Comment:
Or `RawVectors`?

Reply:
Not sure. `Cholesky` uses `RawU` for the equivalent accessor. Though that is more obvious to people who have not read the documentation and are not aware of the conventional naming of the eigenvector-containing matrix. I can see `RawVectors() Matrix` being similarly confusing to some users though. On balance, I think `RawQ` is probably the right approach.

Reply:
> ... conventional naming of the eigenvector-containing matrix.

I tried to check some literature and there doesn't seem to be a consistent notation for the factorization, unlike QR or LQ. I used Q to denote an orthogonal matrix for consistency with these two factorizations but Lapack itself uses Z: https://www.netlib.org/lapack/lug/node30.html For the caller the vectors are the focus, so it makes sense to use `RawVectors`, also to have a pair name for VectorsTo, but as you write it can be confusing to some users and the name is a bit too similar to `RawVector`.